test/functional/lua/fs_spec.lua

```diff
@@ -454,4 +454,74 @@ describe('vim.fs', function()
       end)
     end)
   end)
+
+  describe('is_relative_to()', function()
+    local function is_rel(path, root)
+      return exec_lua([[return vim.fs.is_relative_to(...)]], path, root)
+    end
+    it('works', function()
+      local p = 'a/b'
+      eq(true, is_rel(p, 'a/'))
+      eq(true, is_rel(p, 'a/'))
+      eq(true, is_rel(p, 'a/b'))
+      eq(false, is_rel(p, 'c'))
```

Comment:
nit: `ok` and `ok(not ...)` look more readable. Or `assert`.
```suggestion
      ok(is_rel(p, 'a/b'))
      eq(not is_rel(p, 'c'))
```